<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="SoftwareManagement.sm_apt_install_download_path" skipped="0" tests="1" time="84.71" timestamp="2022-04-28T13:18:30">
	<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED"/>
		<system-out>13:17:05 INFO  ==============================================================
13:17:05 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
13:17:05 INFO  Dir:   software_management_end_to_end/
13:17:05 INFO  Title: Install package to directory specified in tedge config set tmp.path
13:17:05 INFO  ==============================================================
13:17:06 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
13:17:06 INFO  EnvironmentC8y Setup
13:17:07 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
13:17:21 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:17:22 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
13:17:22 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
13:17:23 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
13:17:25 INFO  State of current operation: FAILED
13:17:25 INFO  Assert that (False == value) with value=False ... passed
13:17:26 INFO  Executed sudo, exit status 0
13:17:26 INFO  Executed sudo, exit status 0
13:17:27 INFO  Executed sudo, exit status 0
13:17:31 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:17:31 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
13:17:31 INFO  Delaying execution by 8 seconds
13:17:53 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
13:17:53 INFO  Response status: 201
13:17:53 INFO  Response to action: {
    &quot;creationTime&quot;: &quot;2022-04-28T13:17:53.249Z&quot;,
    &quot;deviceName&quot;: &quot;m32sd10b&quot;,
    &quot;deviceId&quot;: &quot;17884833&quot;,
    &quot;self&quot;: &quot;https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17910986&quot;,
    &quot;id&quot;: &quot;17910986&quot;,
    &quot;status&quot;: &quot;PENDING&quot;,
    &quot;description&quot;: &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;,
    &quot;c8y_SoftwareUpdate&quot;: [
        {
            &quot;name&quot;: &quot;rolldice&quot;,
            &quot;action&quot;: &quot;install&quot;,
            &quot;id&quot;: &quot;5152439&quot;,
            &quot;version&quot;: &quot;::apt&quot;,
            &quot;url&quot;: &quot;https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463&quot;
        }
    ]
}
13:17:53 INFO  Started operation: {'creationTime': '2022-04-28T13:17:53.249Z', 'deviceName': 'm32sd10b', 'deviceId': '17884833', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/17910986', 'id': '17910986', 'status': 'PENDING', 'description': &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;, 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
13:17:53 WARN  , retrying in 0.5 seconds...
13:17:53 WARN  , retrying in 0.5 seconds...
13:17:54 WARN  , retrying in 0.5 seconds...
13:17:54 WARN  , retrying in 0.5 seconds...
13:17:55 WARN  , retrying in 0.5 seconds...
13:17:55 WARN  , retrying in 0.5 seconds...
13:17:56 WARN  , retrying in 0.5 seconds...
13:17:56 WARN  , retrying in 0.5 seconds...
13:17:57 WARN  , retrying in 0.5 seconds...
13:17:57 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py&quot;, line 1294, in __call__
    self.testObj.execute()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py&quot;, line 232, in fun
    return caller(func, *(extras + args), **kw)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 74, in retry_decorator
    logger)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 33, in __retry_internal
    return f()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 28, in assert_install_in_download_path
    assert &quot;rolldice&quot; in os.listdir(f&quot;{install_directory}&quot;)
AssertionError
13:17:57 WARN  AssertionError: ... blocked 
13:17:57 INFO  
13:17:57 INFO  cleanup:
13:17:58 INFO  Executed sudo, exit status 0
13:17:59 INFO  Executed sudo, exit status 0
13:18:03 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:18:03 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
13:18:03 INFO  Delaying execution by 8 seconds
13:18:25 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
13:18:29 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
13:18:30 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
13:18:30 INFO  
13:18:30 INFO  Test duration: 84.71 secs
13:18:30 INFO  Test final outcome:  BLOCKED
13:18:30 INFO  Test outcome reason: AssertionError:
13:18:30 INFO  
</system-out>
	</testcase>
</testsuite>
